src/main/java/net/forgecraft/services/ember/Main.java

```diff
@@ -2,32 +2,64 @@
 
 
 import net.forgecraft.services.ember.app.Services;
+import net.forgecraft.services.ember.bot.listener.ModApprovalListener;
+import net.forgecraft.services.ember.bot.listener.ModUploadListener;
+import net.forgecraft.services.ember.db.DatabaseManager;
 import org.javacord.api.DiscordApi;
 import org.javacord.api.DiscordApiBuilder;
 import org.javacord.api.entity.intent.Intent;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
 
 public final class Main {
     private static final Logger LOGGER = LoggerFactory.getLogger(Main.class);
 
-    private static Main INSTANCE;
+    @Deprecated
+    public static volatile Main INSTANCE;
 
     private final Services services;
     private final DiscordApi discordApi;
 
     public static void main(String[] args) {
         LOGGER.info("Starting application...");
-        INSTANCE = new Main(args);
+
+        var opts = CommandLine.populateSpec(Main.Cli.class, args);
```

Comment:
using an off-the-shelf library is better than reinventing the wheel, especially with something as complex as CLI argument parsing.

I picked this CLI library because it does not have any of the weird and dated conventions that others have, the rest is just preference.